lib/reporter/results.js

```diff
@@ -406,6 +441,11 @@ module.exports = class Results {
     return this;
   }
 
+  setCurrentSection(testcase) {
+    this.currentSectionName = testcase.testName;
```

Comment:
isn't this same as `this.currentTestName` is this extra parameter needed?

Reply:
If we use `this.currentTestName`, then it breaks the current reports. That is because we are adding new pseudo tests for hooks like `__before_hook` / `__after_hook`